runall.sh

```diff
@@ -0,0 +1,97 @@
+fi
+echo DurablePersonNGTest is complete!
+
+echo Running NonVolatileMemAllocatorNGTest...
+mvn -Dtest=NonVolatileMemAllocatorNGTest test -pl mnemonic-core -DskipTests=false > testlog/NonVolatileMemAllocatorNGTest.log
+if [ $? -gt 0 ]
+then
+echo [ERROR] This test case requires 'pmalloc' memory service to pass, please check if 'pmalloc' has been configured correctly! If 'pmalloc' is installed, please refer to testlog/NonVolatileMemAllocatorNGTest.log for detailed information.
+exit 1
+fi
+echo NonVolatileMemAllocatorNGTest is complete!
+
+echo Running VolatileMemAllocatorNGTest...
+mvn -Dtest=VolatileMemAllocatorNGTest test -pl mnemonic-core -DskipTests=false >testlog/VolatileMemAllocatorNGTest.log
+if [ $? -gt 0 ]
+then
+echo [ERROR] This example requires 'vmem' memory service to run, please check if 'vmem' has been configured correctly! If 'vmem' is installed, please refer to testlog/VolatileMemAllocatorNGTest.log for detailed information.
+exit 1
+fi
+echo VolatileMemAllocatorNGTest is complete!
+
+echo Running MemClusteringNGTest...
+mvn -Dtest=MemClusteringNGTest test -pl mnemonic-core -DskipTests=false > testlog/MemClusteringNGTest.log
+if [ $? -gt 0 ]
+then
+echo [ERROR] This example requires 'vmem' memory service to run, please check if 'vmem' has been configured correctly! If 'vmem' is installed, please refer to testlog/MemClusteringNGTest.log for detailed information.
+exit 1
+fi
+echo MemClusteringNGTest is complete!
+
+echo Running DurableSinglyLinkedListNGTest...
+mvn -Dtest=DurableSinglyLinkedListNGTest  test -pl mnemonic-collections -DskipTests=false > testlog/DurableSinglyLinkedListNGTest.log
+if [ $? -gt 0 ]
+then
+echo [ERROR] This test case requires 'pmalloc' memory service to pass, please check if 'pmalloc' has been configured correctly! If 'pmalloc' is installed, please refer to testlog/DurableSinglyLinkedListNGTest.log for detailed information.
+exit 1
+fi
+echo DurableSinglyLinkedListNGTest is complete!
+
+echo Running DurablePersonNGTest...
+mvn -Dtest=DurablePersonNGTest  test -pl mnemonic-collections -DskipTests=false > testlog/DurablePersonNGTest.log
+if [ $? -gt 0 ]
+then
+echo [ERROR] This test case requires 'pmalloc' memory service to pass, please check if 'pmalloc' has been configured correctly! If 'pmalloc' is installed, please refer to testlog/DurablePersonNGTest.log for detailed information.
+exit 1
+fi
+echo DurablePersonNGTest is complete!
+
+echo Running DurableSinglyLinkedListNGPrintTest...
+mvn -Dtest=DurableSinglyLinkedListNGPrintTest test -pl mnemonic-computing-services/mnemonic-utilities-service -DskipTests=false > testlog/DurableSinglyLinkedListNGPrintTest.log
+if [ $? -gt 0 ]
+then
+echo [ERROR] This test case requires 'pmalloc' memory service to pass, please check if 'pmalloc' has been configured correctly! If 'pmalloc' is installed, please refer to testlog/DurableSinglyLinkedListNGPrintTest.log for detailed information.
+exit 1
+fi
+echo DurableSinglyLinkedListNGPrintTest is complete!
+rm -rf testlog
```

Comment:
Please keep this testlog because some user want to have a log to confirm the run pass by theirself.